sceptre/cli/create.py

```diff
@@ -10,9 +11,14 @@
 @click.argument("path")
 @click.argument("change-set-name", required=False)
 @click.option("-y", "--yes", is_flag=True, help="Assume yes to all questions.")
+@click.option(
+    "--disable-rollback/--enable-rollback",
+    default=None,
+    help="Disable the auto rollback and keep resources successfully created or updated",
```

Comment:
This `help` isn't completely accurate any more, since this can also be `--enable-rollback`, which doesn't disable it. I think all these CLI options here and below for disable/enable need to be updated along these lines.

Reply:
fixed